.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+version: 2
+
+
+defaults:
+
+  workspace_root: &workspace_root
+    ~/build
+
+  container: &default_container
+    working_directory: *workspace_root
+    docker:
+      - image: circleci/openjdk:8-jdk
+
+
+jobs:
+
+  build_connectors:
+    <<: *default_container
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - connectors-dependencies-{{ checksum "build.gradle" }}
+            - connectors-dependencies
+          paths:
+            - ~/.gradle/caches/
+            - ~/.gradle/wrapper/
+      - run:
+          name: Test and build connectors project
+          command: ./gradlew -p ./connectors clean build
+      - save_cache:
+          key: connectors-dependencies-{{ checksum "build.gradle" }}
+          paths:
+            - ~/.gradle/caches/
+            - ~/.gradle/wrapper/
+
+
+  build_streams-api:
+    <<: *default_container
```

Comment:
Every project is built inside the `~/build` directory. Is it inside container, or it is the same one for all artifacts and binaries? If it is the second case, then how binaries and artifacts are separated before packaging into container?